src/adtechmedia-serveroptions.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Adtechmedia_Request
+ *
+ * @category File
+ * @package  Adtechmedia_Plugin
+ * @author   yamagleb
+ */
+
+/**
+ * Class Adtechmedia_ServerOptions
+ */
+class Adtechmedia_ServerOptions {
+
+	/**
+	 * Set server options
+	 */
+	static public function set_options() {
+		if ( self::is_apache() ) {
+			self::set_apache_config();
+		} elseif ( self::is_nginx() ) {
+			self::set_nginx_config();
+		}
+	}
+
+	/**
+	 * Delete server options
+	 */
+	static public function delete_options() {
+		if ( self::is_apache() ) {
+			self::delete_apache_config();
+		} elseif ( self::is_nginx() ) {
+			self::delete_nginx_config();
+		}
+	}
+
+	/**
+	 * Returns true if server is Apache
+	 *
+	 * @return boolean
+	 */
+	static public function is_apache() {
+		// Assume apache when unknown, since most common.
+		if ( empty( $_SERVER['SERVER_SOFTWARE'] ) ) {
+			return true;
+		}
+
+		return isset( $_SERVER['SERVER_SOFTWARE'] ) && stristr( sanitize_text_field( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ) ), 'Apache' ) !== false;
+	}
+
+	/**
+	 * Returns true if server is nginx
+	 *
+	 * @return boolean
+	 */
+	static public function is_nginx() {
+		return isset( $_SERVER['SERVER_SOFTWARE'] ) && stristr( sanitize_text_field( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ), 'nginx' ) ) !== false;
+	}
+
+	/**
+	 * Set config for Apache
+	 */
+	static public function set_apache_config() {
+		$path     = explode( 'wp-content', dirname( __FILE__ ) )[0] . '/.htaccess';
```

Comment:
- path to `.htaccess` file -> `explode( 'wp-content', dirname( __FILE__ ) )[0] . '/.htaccess'` can be moved into separate class method to avoid duplication.